src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -56,12 +52,20 @@ class ValidateActor(wdlSource: WdlSource, workflowInputs: Option[WdlJson])
     }
   }
 
+  private def validateRuntimeOptions(call: Call,  workflowOptions: Option[String]): Try[CromwellRuntimeAttributes] = {
+    workflowOptions match {
+      case Some(wo) =>
+        WorkflowOptions.fromJsonString(wo) map { options => CromwellRuntimeAttributes(call.task.runtimeAttributes, options, CromwellBackend.backend().backendType) }
+      case None => Try(CromwellRuntimeAttributes(call.task.runtimeAttributes, CromwellBackend.backend.backendType))
+    }
+  }
+
   private def validateWorkflow(sentBy: ActorRef): Unit = {
     logger.info(s"$tag for $sentBy")
     val futureValidation: Future[Unit] = for {
       namespaceWithWorkflow <- Future(NamespaceWithWorkflow.load(wdlSource))
       validatedInputs <- validateInputs(namespaceWithWorkflow, workflowInputs)
-      runtime = namespaceWithWorkflow.workflow.calls foreach { _.toRuntimeAttributes }
+      validatedRuntimeOptions <- Future.fromTry(TryUtil.sequence(namespaceWithWorkflow.workflow.calls map { call => validateRuntimeOptions(call, workflowOptions) }))
```

Comment:
:pensive:

However I think this might be the spot where I said the other day the surrounding stuff was so baked in that you might as well